include/rcutils/error_handling.h

```diff
@@ -89,6 +89,35 @@ void
 rcutils_set_error_state(
   const char * error_msg, const char * file, size_t line_number, rcutils_allocator_t allocator);
 
+/// Check an argument for a null value.
+/**
+ * If the argument's value is null, set the error message saying so and
+ * return the error_return_type after setting the error message.
+ *
+ * \param[in] argument The argument to test.
+ * \param[in] error_return_type The type to return in case of an error.
+ * \param[in] allocator The allocator to use if an error message needs to be allocated.
+ */
+#define RCUTILS_CHECK_ARGUMENT_FOR_NULL(argument, error_return_type, allocator) \
+  RCUTILS_CHECK_FOR_NULL_WITH_MSG(argument, #argument " argument is null", \
+    return error_return_type, allocator)
+
+/// Check an value for a null value with a user defined message and error statement.
```

Comment:
I think keeping "argument" in the description here is fine, it'll just refer to the macro argument instead of the function's argument.
It'll also stay consistent with the description below that refers to it as an "argument"